benchmarks/automated/tests/utils/ts.py

```diff
@@ -154,6 +154,7 @@ def stop_recording_docker_stats(self, model_name, num_workers, batch_size):
         self.connection.run("ps axl|grep -e '--no-stream'| grep -v color | awk '{print $3}' | xargs kill -9", warn=True)
         self.connection.run(f"cp nohup.out nohup.{model_name}.{num_workers}.{batch_size}", warn=True)
         self.connection.run(f"rm nohup.out", warn=True)
+        time.sleep(3)
```

Comment:
Does this code run in CI? Slightly worried about timeouts

Reply:
It does not run in CI (i.e. PR build checks), it does run within codebuild jobs that are configured internally at AWS to populate cloudwatch metrics (which simply run the ci/benchmark/buildspec.yml) - so no surprises. 

The CI should not be affected by this.